lib/iris/fileformats/pp.py

```diff
@@ -2129,8 +2141,92 @@ def save(cube, target, append=False, field_coords=None):
         verify_rules_ran = rules_result.matching_rules
 
         # Log the rules used
-        iris.fileformats.rules.log('PP_SAVE', target if isinstance(target, basestring) else target.name, verify_rules_ran)
+        # XXX Get rid of this? Or have an optional `target` parameter?
+        #iris.fileformats.rules.log('PP_SAVE', target if isinstance(target, basestring) else target.name, verify_rules_ran)
```

Comment:
We need to keep this. (We can consider deprecating the logging mechanism as a separate activity.)